src/components/comments/container/commentsContainer.js

```diff
@@ -122,14 +121,20 @@ class CommentsContainer extends Component {
     return parent;
   };
 
-  _getComments = () => {
-    const { author, permlink } = this.props;
+  _getComments = async () => {
+    const {
+      author,
+      permlink,
+      currentAccount: { name },
+    } = this.props;
 
-    getComments(author, permlink).then(comments => {
-      this.setState({
-        comments,
-      });
-    });
+    await getComments(author, permlink, name)
+      .then(async comments => {
```

Comment:
can you remove async keyword, because you didnt use await inside of this function